app/src/main/java/com/example/next_show/fragments/SavedFragment.java

```diff
@@ -60,7 +60,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         savedShows = new ArrayList<>();
 
         // initialize the adapter
-        adapter = new ShowAdapter(getActivity(), savedShows, new NavigateSavedToDetail());
+        adapter = new ShowAdapter(getActivity(), savedShows, new NavigateSavedToDetail(), null);
```

Comment:
Yeah, I would just entirely remove passing in the filter to the adapter.  Just tell the adapter "this is the list of shows to display" and hide the knowledge of page specific filtering from it